### ISSUE TITLE
Migrate old-style collections to triplestore

### ISSUE THREAD

**Issue by jgonggrijp** [CentreForDigitalHumanities/EDPOP#226]
I merged #181, started working on #224 and then realized that all my collections were gone. Or rather, they don't appear in the frontend anymore because my old testing collections are stored in the relational database while the backend now attempts to retrieve them from the triplestore.

This needs to be addressed before we can deploy from the `develop` branch.

**Comment by lukavdplas** [comment]
There is a [convert_legacy_data](https://github.com/UUDigitalHumanitieslab/EDPOP/tree/develop/backend/triplestore/convert_legacy_data) module in the backend which defines functions for this, but it's not used.

I believe the reason why collections are not migrated in #181 was the interdependencies between records, collections, and annotations; this makes it difficult to migrate one model but not the other.

**Comment by jgonggrijp** [comment]
Makes sense. That does mean we have to implement migrations for all three before we can deploy off `develop` again.

**Comment by jgonggrijp** [comment]
@tijmenbaarda I think we can close this as of #311 and #324?

**Comment by tijmenbaarda** [comment]
I think so too!
